files-community/Files
# Feature: Added support for restoring tabs after restarting Windows

**Resolved / Related Issues**
   Closes #12624

**Validation**
How did you test these changes?
   1. Open multiple Files instances, and open several tabs for each instance.(Need to check "Continue where you left off")
   2. Modify tabs at will, including opening tabs, closing tabs, changing tab paths, etc.
   3. Check if "LastAppsTabsWithIDList" in user_settings.json is consistent with the open tab page.
   4. Restart the computer directly.
   5. After restarting, open Files and see if the tab status before shutdown is automatically restored.

**Screenshots (optional)**
Add screenshots here.
